tests/kokkos-based/mdspan_to_view.cpp

```diff
@@ -0,0 +1,81 @@
+
+#include "gtest_fixtures.hpp"
+#include "helpers.hpp"
+
+template<class MDSpanValueType, class KViewValueType = MDSpanValueType>
+void mdspan_to_view_test_impl()
+{
+  using std::experimental::mdspan;
+  using std::experimental::extents;
+  using std::experimental::dynamic_extent;
+
+  // rank1, non-const
+  {
+    std::vector<MDSpanValueType> a(5);
+    using mdspan_t = mdspan<MDSpanValueType, extents<dynamic_extent>>;
+    mdspan_t mdsp(a.data(), a.size());
+
+    auto kv = KokkosKernelsSTD::Impl::mdspan_to_view(mdsp);
+    using kv_type = decltype(kv);
+    static_assert(kv_type::rank == 1);
+    static_assert(std::is_same_v<typename kv_type::value_type, KViewValueType>);
+    EXPECT_TRUE(kv.extent(0) == 5);
```

Comment:
(OPTIONAL) Would you consider also adding the following tests to each case?

1. `EXPECT_EQ(mdsp.data(), kv.data())` (check that it's really a shallow copy)
2. Whether the `const`-ness of the Kokkos::View's element type is as expected (I think you do that below)

Thanks!

Reply:
done!